konnectivity-client/go.mod

```diff
@@ -3,8 +3,8 @@ module sigs.k8s.io/apiserver-network-proxy/konnectivity-client
 go 1.17
 
 require (
-	github.com/golang/protobuf v1.4.3
-	google.golang.org/grpc v1.42.0
+	github.com/golang/protobuf v1.4.1
```

Comment:
https://github.com/kubernetes/kubernetes/blob/master/go.mod#L47 shows v1.5.2. Would it make sense to make that our version? (Ditto https://github.com/kubernetes/kubernetes/blob/release-1.23/go.mod)